### DISCUSSION TITLE
Why did everything on the site turn gray, even the video?

### DISCUSSION THREAD

**Discussion by DenisVASI9** [react-hook-form/react-hook-form#1762] [General] [answered]
Now it's just impossible to distinguish the elements from each other.

**Comment by bluebill1049** [comment, marked as answer]
https://www.google.com/search?q=black+live+matter&oq=black&aqs=chrome.0.69i59j69i57j46j0j69i61j69i60l3.817j0j7&sourceid=chrome&ie=UTF-8
